network-api/networkapi/settings.py

```diff
@@ -722,3 +730,36 @@
         "127.0.0.1",
         "10.0.2.2",
     ]
+
+# Django Pattern Library
+# https://torchbox.github.io/django-pattern-library/
+#
+# Pattern library isn’t intended for production usage, and hasn’t received
+# extensive security scrutiny. Don't enable it on production.
+#
+# PATTERN_LIBRARY_ENABLED is set to True in docker-compose.yml for local development.
+# For pattern library to work with CSP, you also need to add the following to your .env file:
+#    PATTERN_LIBRARY_ENABLED=1
+#    X_FRAME_OPTIONS=SAMEORIGIN
+#    CSP_FRAME_ANCESTORS="'self'"
```

Comment:
@jhonatan-lopes I see your comments about the CSP fixes. I have added the instructions on the settings file now. Is this where you have in mind to add the documentation?

Reply:
This is perfect, thanks!